blueman/Functions.py

```diff
@@ -131,20 +141,20 @@ def launch(cmd, paths=None, system=False, icon_name=None, sn=True, name="blueman
         context.set_icon(icon)
 
     appinfo = Gio.AppInfo.create_from_commandline(cmd, name, flags)
-    launched = appinfo.launch(files, context)
+    launched: bool = appinfo.launch(files, context)
 
     if not launched:
         logging.error("Command: %s failed" % cmd)
 
     return launched
 
 
-def setup_icon_path():
+def setup_icon_path() -> None:
     ic = Gtk.IconTheme.get_default()
     ic.prepend_search_path(ICON_PATH)
 
 
-def get_icon(name, size=24, fallback="image-missing"):
+def get_icon(name: str, size: int = 24, fallback: str = "image-missing") -> GdkPixbuf.Pixbuf:
```

Comment:
Unrelated but I immediately started wondering if we actually use it. We do but only in get_notification_icon which is also unused. I'll open a PR to remove them later.